apps/passport-server/test/pretix/mockDevconnectPretixApi.ts

```diff
@@ -0,0 +1,97 @@
+import {
+  DevconnectPretixConfig,
+  DevconnectPretixEvent,
+  DevconnectPretixItem,
+  DevconnectPretixOrder,
+  IDevconnectPretixAPI,
+} from "../../src/apis/devconnectPretixAPI";
+import { logger } from "../../src/util/logger";
+import {
+  DevconnectPretixDataMocker,
+  EVENT_A,
+  EVENT_B,
+  EVENT_C,
+  IMockDevconnectPretixData,
+  ITEM_1,
+  ITEM_2,
+  ORG_CONFIG_ID,
+} from "./devconnectPretixDataMocker";
+
+export const MOCK_PRETIX_API_CONFIG: DevconnectPretixConfig = {
```

Comment:
i recommend creating this in the mocker file rather than in the api file

Reply:
discussed offline that we can have a separate config mocker for this -- punting to a future PR